placeSpecificContent/theSpot/moneyTree/entityScripts/moneyTreeProximityClient.js

```diff
@@ -0,0 +1,100 @@
+// moneyTreeProximityClient.js
+//
+//  Created by Mark Brosche on 10-18-2018
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* global EventBridge Users AccountServices Agent Avatar */
+
+(function(){
+    var SECRETS = Script.require(Script.resolvePath('../moneyTreeURLs.json')),
```

Comment:
this is dangerous because it is running on a client script accessible to anyone. hopefully not a huge deal.